registry/server/auth.ts

```diff
@@ -318,10 +318,14 @@ export default async (app: Express, settingsService: SettingsService, config: an
 };
 
 async function getEntityWithCreds(provider: string, identifier: string, secret: string | null): Promise<User | null> {
-    const user = await db.select().from('auth_entities').first('identifier', 'id', 'role', 'secret').where({
-        provider,
-        identifier,
-    });
+    const user = await db
+        .select()
+        .from('auth_entities')
+        .first('identifier', 'id', 'role', 'secret')
+        .where({
+            provider,
+        })
+        .andWhereRaw('LOWER(identifier) = LOWER(?)', [identifier]);
```

Comment:
won't it break mysql?
Also is it effective?

Reply:
Tests on sqlite and mysql pass
Additional test for case-insensitive identifier added.

Reply:
Regarding effectiveness https://stackoverflow.com/a/7005332/5825564
Can add an index on lower(identity column) though I'm not sure that this query and table is used very much.

Reply:
Ok, lets see how it will affect DB load